docs/versions.md

```diff
@@ -21,26 +21,34 @@ server = TSC.Server('http://SERVER_URL')
 print(server.version)
 ```
 
-For example, the code might display version `2.3`.
+For example, the code might display version `2.3`. 
 
-## Use another version of the REST API
+## Use the REST API version supported by the server
 
-To use another version of the REST API, set the version like so:
+To always use the latest version of the REST API that is supported by the instance of Tableau Server you are connecting to, set the version using the `use_server_version()` method:
 
 ```py
 import tableauserverclient as TSC
 
 server = TSC.Server('http://SERVER_URL')
 
-<<<<<<< HEAD
-server.version = 2.4
-=======
+server.use_server_version()
 
-server.version = '2.6'
+```
 
+## Use a specific version of the REST API
+
+To use another version of the REST API, set the version like so:
+
+```py
+import tableauserverclient as TSC
+
+server = TSC.Server('http://SERVER_URL')
+
+ 
+server.version = '2.6'
 
 
->>>>>>> 87acd20... Update versions.md (#195)
```

Comment:
There is a conflict here.  You need to resolve that before we push this.

Reply:
Resolved.